minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -146,29 +142,29 @@ public boolean isServing() {
   }
 
   /**
-   * Stops the master and cleans up client connections.
+   * Stops the master processes and cleans up client connections.
    */
   public void stop() throws Exception {
-    // This shutdown needs to be done in a loop with retry because the interrupt signal can
-    // sometimes be ignored in the master implementation. For example, if the master is doing
-    // a hdfs listStatus RPC (hadoop version is 1.x), the interrupt signal is not properly handled.
-    while (mMasterThread.isAlive()) {
```

Comment:
I think we need to keep this. If we have shutdown code, then I think it should work instead of doing something that can potentially broken. This was added when I debug MasterFaultToleranceIntegrationTest with hdfs profile.